modules/atomic-widgets/styles/styles-renderer.php

```diff
@@ -6,27 +6,26 @@
 use Elementor\Modules\AtomicWidgets\PropsResolver\Props_Resolver;
 
 class Styles_Renderer {
+	const DEFAULT_SELECTOR_PREFIX = '.elementor ';
```

Comment:
```suggestion
	const DEFAULT_SELECTOR_PREFIX = '.e ';
```

I think I would also remove the space at the end. it's kinda confusing, WDYT?

Reply:
Done